### PR TITLE
Document common GitHub topic for plugins

### PR DESCRIPTION
**Pull request by nrb**
This is similar to how Octant documents its plugins using a GitHub feature.

### CODE REVIEW

File: site/docs/master/overview-plugins.md [hunk 1]
```diff
@@ -19,6 +19,9 @@ Anyone can add integrations for any platform to provide additional backup and vo
 
 After you publish your plugin on your own repository, open a PR that adds a link to it under the appropriate list of [supported providers][3] page in our documentation.
 
+You can also add the `velero-plugin` GitHub Topic to your repo, and it be shown under the aggregated list of repositories automatically.
```
Comment (skriss) [inline site/docs/master/overview-plugins.md]: "...and it **will** be shown..."

Comment (skriss) [inline site/docs/master/overview-plugins.md]: will it actually be shown automatically?

File: site/docs/master/overview-plugins.md [hunk 2]
```diff
@@ -19,6 +19,9 @@ Anyone can add integrations for any platform to provide additional backup and vo
...
+
 [1]: https://github.com/vmware-tanzu/velero-plugin-example/
 [2]: custom-plugins.md
-[3]: supported-providers.md
\ No newline at end of file
+[3]: supported-providers.md
+[4]: https://github.com/topics/velero-plugin
```
Comment (skriss) [inline site/docs/master/overview-plugins.md]: were you intending to use this link somewhere?

**Comment by skriss** [comment]
:+1: on establishing/documenting the topic label!

File: site/docs/master/overview-plugins.md (hunk 2 above)
Comment (nrb) [reply site/docs/master/overview-plugins.md]: D'oh, yes!

File: site/docs/master/overview-plugins.md (hunk 1 above)
Comment (nrb) [reply site/docs/master/overview-plugins.md]: Yeah, the topic URL in the links below is basically a search for all repos mentioning it, so it should be automatic, possibly some slight delays.

Comment (skriss) [reply site/docs/master/overview-plugins.md]: Ah, I see what you mean. :+1:

**Pull request merged**